README.md

```diff
@@ -16,7 +16,41 @@ TOC:
 
 # VM
 
-TODO
+Prerequesities:
+- A **SQL Server 2017 Enterprise on Red Hat Enterprise Linux 7.4 (RHEL)** VM
+- Two "Inbound port rule" on the associated "Azure Network Security Group", one for the port 1433 and the other for the port 88 to allow external connections to the web app and to the database endpoint.
+- On your local machine, a "[SQL Operations Studio](https://docs.microsoft.com/en-us/sql/sql-operations-studio/download?)" installed
+
+*Note: for more details about the manual installation of SQL Server 2017 over Red Hat Enterprise Linux, please see [here](https://docs.microsoft.com/en-us/sql/linux/quickstart-install-connect-red-hat?view=sql-server-linux-2017).*
+
+## Database
+
+From within the RHEL74 VM, run the following command:
+```
+cd TODO
```

Comment:
@mturcottemsft could you confirm this .bak file download has to be done from within the VM? Furthermore, in which folder it has to be downloaded?

Reply:
Le script restore.sql s'attend à trouver le backup ici: "/var/opt/mssql/backup/WideWorldImporters-Full.bak".

Cela dit, ta version de la doc parle d'exécuter le restore.sql avant le script init-db.sql (qui devrait plutôt être init-db.sh).

En fait, seul le script init-db.sh doit être exécuter. Il s'occupe de créer le répertoire backup sous /var/opt/mssql/ pour ensuite télécharger dans ce répertoire (après un 'cd backup/') et exécuter le restore.sql


À suivre… mon meeting m'appele.